pkg/adapterManager/registry.go

```diff
@@ -20,20 +20,19 @@ import (
 	"github.com/golang/glog"
 	"istio.io/mixer/pkg/adapter"
 	"istio.io/mixer/pkg/aspect"
-	"istio.io/mixer/pkg/config"
 )
 
 // registry implements pkg/adapter/Registrar.
 // registry is initialized in the constructor and is immutable thereafter.
 // All registered builders must have unique names.
```

Comment:
Update this comment -> "All registered builders must have a unique name per aspect kind"